hack/preview.sh

```diff
@@ -36,10 +38,16 @@ git checkout -b $PREVIEW_BRANCH
 # this needs to be pushed to your fork to be seen by argocd
 $ROOT/hack/util-set-development-repos.sh $MY_GIT_REPO_URL development $PREVIEW_BRANCH
 
+
 # set the API server which SPI uses to authenticate users to empty string (by default) so that multi-cluster
 # setup is not needed
 $ROOT/hack/util-set-spi-api-server.sh "$SPI_API_SERVER"
 
+if [ -n "$OVERLAY" ]; then
+    echo "Configuring E2E overlay"
+    $ROOT/hack/util-set-development-repos.sh $MY_GIT_REPO_URL $OVERLAY $PREVIEW_BRANCH
```

Comment:
not needed

Reply:
hehe... That's the main reason I converted this PR to draft... After sending it I noticed this part I didn't clean up properly :-D